src/features/advanced/inv-compress-inventory-info.ts

```diff
@@ -0,0 +1,77 @@
+import { applyScopedClassCssRule } from '@src/infrastructure/prun-ui/refined-prun-css';
+import classes from './inv-compress-inventory-info.module.css';
+import css from '@src/utils/css-utils.module.css';
+import ContextControls from '@src/components/ContextControls.vue';
+import { getInvStore } from '@src/core/store-id';
+import { sitesStore } from '@src/infrastructure/prun-api/data/sites';
+import {
+  getEntityNaturalIdFromAddress,
+  getLocationLineFromAddress,
+} from '@src/infrastructure/prun-api/data/addresses';
+import { warehousesStore } from '@src/infrastructure/prun-api/data/warehouses';
+import { shipsStore } from '@src/infrastructure/prun-api/data/ships';
+
+async function onTileReady(tile: PrunTile) {
+  if (!tile.parameter || tile.command === 'SHPI') {
```

Comment:
```suggestion
  if (!tile.parameter) {
```

You don't need to check for `SHPI` since you only observe `INV` tiles:
```ts
tiles.observe('INV', onTileReady);
```